### ISSUE TITLE
Keyboard short cut Ctrl+shift+F8 on linux does not insert citation and zooms the presentation tab

### ISSUE THREAD

**Issue by astayleraz** [rstudio/rstudio#10027]
### System details

    RStudio Edition : 
    RStudio Version : 2022.01.1-daily-171.pro1
    OS Version      : Ubuntu
    R Version       : 

### Steps to reproduce the problem

Automation failure: https://build.rstudioservices.com/job/IDE/job/qa-opensource-automation/903/testReport/rstudio_server_pro.tests.test_Citations/TestCitations/test_no_duplicate_citations/

Note: This was not an issue in 2021.12.0-daily+64.pro1 and the test was passing on this build. Not sure if this might be related to the work done in shortcuts in https://github.com/rstudio/rstudio/issues/9915

Test steps:
1. Create a markdown file in a ubuntu or linux instance.
2. Go into visual mode.
3. Use hotkey to insert citation (Ctrl+shift+F8)

### Describe the problem in detail
Actual:
Zooms the presentation tab.

Looking at the modify keyboard shortcuts modal it looks like zoom presentation was newly added and shares the same hotkeys as insert citation.

![Screen Shot 2021-10-29 at 2 11 51 PM](https://user-images.githubusercontent.com/1482677/139502370-53ba5dd8-b1fa-4e07-924f-2920503f42a8.png)

![Screen Shot 2021-10-29 at 2 12 19 PM](https://user-images.githubusercontent.com/1482677/139502389-e7b008cf-1a75-4866-b997-259114d82a6d.png)

### Describe the behavior you expected
Expected:
Should bring up citation modal and not zoom the presentation tab. I guess if we use the hotkey for both tabs, the citation hotkey would only work if the focus is in the source pane or visual editor? I think it's probably better to not share the same hotkey in this case.

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [ ] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by astayleraz** [comment]
This should be fixed here https://github.com/rstudio/rstudio/commit/4c4a64f92cef7c9f30ad5530665dd2a0470e975b

**Comment by astayleraz** [comment]
Verified in 2022.01.0-daily+192.pro1 this is working again as expected and added the check back into the automation test on this. Closing issue.
